op-node/rollup/derive/span_batch_test.go

```diff
@@ -18,6 +18,30 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/testutils"
 )
 
+// initializedSpanBatch creates a new SpanBatch with given SingularBatches.
+// It is used *only* in tests to create a SpanBatch with given SingularBatches as a convenience.
+// It uses default value of 1 for AppendSingularBatch's seqNum.
+// It will also ignore any errors that occur during AppendSingularBatch.
+// Tests should manually set the first bit of the originBits if needed using SetFirstOriginChangedBit
+func initializedSpanBatch(singularBatches []*SingularBatch, genesisTimestamp uint64, chainID *big.Int) *SpanBatch {
+	spanBatch := NewSpanBatch(genesisTimestamp, chainID)
+	if len(singularBatches) == 0 {
+		return spanBatch
+	}
+	for i := 0; i < len(singularBatches); i++ {
+		if err := spanBatch.AppendSingularBatch(singularBatches[i], uint64(1)); err != nil {
```

Comment:
I remember that there was a conversation about this here, but I cannot find it. I suggested setting the sequence to `i` like
```suggestion
		if err := spanBatch.AppendSingularBatch(singularBatches[i], uint64(i)); err != nil {
```
was that deliberately dismissed?

Reply:
I don't remember seeing that suggestion, but I like it and will take it!

Reply:
Then I probably forgot to submit 🙈 the idea behind it is that this would treat the list of batches as all belonging to the same L1 origin/epoch, with the first batch being the first of the epoch.